frontend/src/hooks/useChangeToSkip.js

```diff
@@ -0,0 +1,15 @@
+import { useMutation } from "@tanstack/react-query";
+
+import { PROJECT_ID, queryClient } from "../store";
+import { postConfig } from "../utils/fetch_configs";
+
+const useChangeToSkip = () =>
+    useMutation({
+        mutationFn: ({ dataID, message, oldLabelID }) =>
+            fetch(`/api/modify_label_to_skip/${dataID}/`, postConfig({ dataID, message, oldLabelID })),
```

Comment:
I'm noticing that when you adjudicate from the history table, the little badge above the admin table does not update until a full refresh. 
<img width="804" alt="Screen Shot 2023-05-25 at 11 36 58 AM" src="https://github.com/RTIInternational/SMART/assets/10951569/2218a59d-028a-473a-8ac6-8e54022d8dc2">

In the annotate tab, adjudicating still does update the badge. In the old way, this occurred in the actions via `dispatch(getAdminCounts(projectID));` which would tell the admin counts to refresh.